docs/concepts-throttling.md

```diff
@@ -0,0 +1,167 @@
+dispatchThrottlingOnBatchMessageEnabled | Whether to count messages by entry (batch). By default, it's disabled.<br /><br />Note that setting it to `true` may lead to an inaccurate approximation of total message count but maximize Pulsar's throughput while keeping stable read requests to the bookies. For example, assume you've set the rate limit to `10/s`, if you set `dispatchThrottlingOnBatchMessageEnabled` to `true`, the broker only reads 10 entries and delivers them to the client per second, despite the number of messages per entry. | false
+dispatchThrottlingOnNonBacklogConsumerEnabled | Whether the dispatch throttling on non-backlog consumers is enabled. By default, it's enabled.<br />When it is set to `false`:<br /><li>If all the consumers in one subscription have no backlog, the message dispatch throttling is turned off automatically even if `dispatchThrottlingRateInMsg` and `dispatchThrottlingRateInByte` are configured.</li><li>If at least one consumer has a backlog, the throttling is turned on automatically.</li> | true
+
+:::note
+
+- You can use `dispatchThrottlingRateInMsg` and `dispatchThrottlingRateInByte` simultaneously (logical AND).
+- Ensure that only one of `preciseDispatcherFlowControl` and `dispatchThrottlingOnBatchMessageEnabled` is enabled at one time since they are mutually exclusive. Both parameters can be used to improve the over-delivery issues (see [Limitations](#limitations)). The difference between them is:
+  - When `preciseDispatcherFlowControl` is enabled, Pulsar considers the number of messages per entry. This parameter takes effect when the broker reads entries from the bookies.
+  - When `dispatchThrottlingOnBatchMessageEnabled` is enabled, Pulsar ignores the number of messages per entry. This parameter takes effect when the broker updates the counter after sending messages to the client.
+
+:::
+
+## Limitations
+
+Message dispatch throttling may cause messages over-delivered per unit of time due to the following reasons:
+
+1. **The broker may read more entries or bytes from the bookies than the throttling limit.**
+
+   a) **The byte size of messages delivered to the client may exceed the configured threshold.**
+   
+     When you set the dispatch rate limit in bytes/throttling-period (`dispatchThrottlingRateInByte`/`ratePeriodInSecond`), the broker calculates $$the \ number \ of \ entries \ to \ read \ from \ bookies$$ in one throttling period through the following equation:
+   
+     $$
```

Comment:
I found there are many symbols `$$` and `\`, it seems that they are not valid markdown symbols.

Reply:
@gaoran10 Thanks for noticing it. I should have attached all the preview screenshots to make this clear:)

This is a new style to highlight the unknown of the match equation. The preview looks good as follows.

<img width="1056" alt="image" src="https://user-images.githubusercontent.com/60642177/224476449-9fad61c3-1795-4a1b-94d2-2479f03862e2.png">

Reply:
Oh, got it, thanks.